### DISCUSSION TITLE
✅ Creating an OSPO Local Meetup - Helsinki (English speaking)

### DISCUSSION THREAD

**Discussion by CsatariGergely** [todogroup/ospology#180] [Community Proposals]
Helsinki has a massive amount of IT companies all of them having some relationship to open source. The city deserves its OSPO Local Meetup to share share experiences and connect OSPO people. 

Location would be in the capital area (Helsinki, Espoo, Vantaa) hopefully in office spaces offered by our future sponsors.

**Comment by anajsana** [comment]
Thanks @CsatariGergely for your proposal submission! Our Steering Committee will review and get back to you with a final resolution 🙂 The next SC meeting is October 5th and this topic has been added to the agenda for review. 

Meanwhile, please take a moment to review our [Local Host Guidelines](https://github.com/todogroup/ospology/blob/main/local-meetups/local-host-guidelines.md#todo-local-host-guidelines). The person that decides to become the TODO Local Host of these chapters, must meet the criteria.

**Comment by CsatariGergely** [comment]
Thanks @anajsana . 
I think both myself and the planned co-organiser fits to the requirements. (By the way the list is a bit scrambled up now. Here is a fix https://github.com/todogroup/ospology/pull/183).
I would like to have a meeting to discuss this particular detail if possible (see my mail about it).

**Comment by anajsana** [comment]
Good news @CsatariGergely ! The Steering Committee approved Helshinki's OSPO Local Meetup creation during yesterday's SC monthly meeting 🙌 I'll start creating the registration page via LF Online Community and the slack channel ASAP. Once created, I'll give you full organizer permissions to the chapter

**Reply by CsatariGergely** [reply]
Thanks, very good news indeed.

**Reply by anajsana** [reply]
The registration page has beem created: https://community.linuxfoundation.org/ospo-local-meetup-helsinki/ 
Also, a new slack channel called #chapter-helsinki is available in the TODO slack :)
